### ISSUE TITLE
NVDA remains silent and doesn't announce the corrected word in MS Word 2016 if AutoCorrect is on

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#11032]
### Steps to reproduce:
AutoCorrect is On by default in Microsoft Word, but:
1. To make sure if it's On, go to the File tab, choose Options, go to the Proofing Tab, and select the AutoCorrect Options button. Make sure the Replace text as you type check box is checked.
2. Having returned to the main MS Word window, if you type many misspelled words such as "namme" for "name", "noboddy" for "nobody", "resorces" for "resources", etc., followed by Space, MS Word automatically corrects them.
### Actual behavior:
NVDA remains strangely silent while these corrections are made.
### Expected behavior:
NVDA should alert users that something has been corrected: for example if "namme" with two Ms is corrected it should say: "AutoCorrect: Name, N A M E".
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
alpha-20015,56a62097
#### Windows version:
Windows 10 Pro 64-Bit Version 2004 (OS Build 19041.207)
#### Name and version of other software in use when reproducing the issue:
Microsoft Word 2016
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2019.3.1
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No, as it presumably has no effect here.

**Comment by amirsol81** [comment]
This is also duplicable with MS Word 2019, and NVDA Alpha 20220.

**Comment by amirsol81** [comment]
@seanbudd @michaelDCurran For the record, NVDA Alpha 28043 displays this bug with MS Word 2021.

**Comment by Qchristensen** [comment]
Someone just reported this and I can replicate.  I'm not sure if Word actually exposes anything?  The only visual indicator is if you move focus BACK to that word, there is a small blue underline under the first letter.  If you move the mouse directly over that underline it pops up an "autocorrect options" button which lets you:
- Change back to <text>
- Stop autocorrecting <text>
- Control Autocorrect options
(where <text> is the text it autocorrected)

There is no difference in the regular context menu (if you press the applications key / shift+f10 with the keyboard) compared to a regular correctly spelled word.

I tried with Office 365 (64-bit) Version: 16.0.19725.20126
on Windows 11 (64-bit) Version: 25H2, Build: 26200.7922 
running NVDA Alpha-54879,a94c7f85.exe

With both UIA Default (when necessary) and Always (no difference).

**Comment by bhavyashah** [comment]
I’d add that when I enter “* “ in Word, the star is auto-formatted to a bullet. NVDA does not report this when it happens whereas Narrator does.
